src/util.rs

```diff
@@ -59,17 +97,35 @@ lazy_static! {
 
 pub fn detect_history_files() -> Vec<String> {
     let mut files = Vec::new();
-    let browsers = vec!["safari", "chrome", "firefox", "brave"];
+    let browsers = vec![
```

Comment:
I think we can delete this variable and iterate `DEFAULT_PROFILES` directly

Reply:
something like this?

```rust
pub fn detect_history_files() -> Vec<String> {
    let mut files = Vec::new();
    for p in DEFAULT_PROFILES {
        debug!("detect {p}...");
        if let Ok(entries) = glob::glob(p) {
            for e in entries {
                match e {
                    Ok(file) => files.push(file.into_os_string().into_string().unwrap()),
                    Err(e) => debug!("glob err:{:?}", e),
                }
            }
        }
    }

    files
}
```

Reply:
Yep